VideoWeb/VideoWeb.UnitTests/Controllers/ProfileController/GetUserProfileTests.cs

```diff
@@ -20,13 +22,14 @@ public class GetUserProfileTests
         private ProfilesController _controller;
         private Mock<IUserApiClient> _userApiClientMock;
         private Mock<ILogger<ProfilesController>> _mockLogger;
+        private ClaimsPrincipal claimsPrincipal;
```

Comment:
the naming is off. Should be _claimsPrincipal;

Reply:
updated variable name